var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -124,6 +124,9 @@ class Cmake(Package):
         when="@3.15.5",
     )
 
+    # Ensure CMAKE_INSTALL_RPATH does not error in projects with statically linked binaries
+    patch("rpath-static.patch", when="@3.22.0:3.30")
```

Comment:
If you're trying to avoid bringing that extra binary file you added in the upstream patch (that seems reasonable enough to avoid using the actual upstream patch) can you at least link to your PR and indicate why we're not using the upstream patch?